substrate/frame/benchmarking/src/lib.rs

```diff
@@ -311,6 +311,78 @@ pub use v1::*;
 /// 	}
 /// }
 /// ```
+///
+/// ## Migrate from v1 to v2
+///
+/// If you want to migrate your code from benchmarking v1 to benchmarking v2, you may follow these
+/// steps:
+/// 1. Change the import from `frame_benchmarking::v1::` to `frame_benchmarking::v2::*`, or
+///    `frame::benchmarking::prelude::*` under the umbrella crate;
+/// 2. Move the code inside the v1 `benchmarks! { ... }` block to the v2 benchmarks module `mod
+///    benchmarks { ... }` under the benchmarks macro (`#[benchmarks]` or `#[instance_benchmarks]`);
+/// 3. Turn each v1 benchmark into a function inside the v2 benchmarks module with the same name,
+///    having either with a blank return type or a return type compatible with `Result<(),
+///    BenchmarkError>`. For instance, `foo { ... }` can become `fn foo() -> Result<(),
+///    BenchmarkError>`. More in detail:
+///   3.1 Move all the v1 complexity parameters as [ParamRange](`v2::ParamRange`) arguments to the
+/// v2 function, and their setup code to the body of the function. For instance, `let y in 0 .. 10
```

Comment:
indentation here looks a messy.

Reply:
True, fixed in https://github.com/paritytech/polkadot-sdk/pull/6093/commits/d80beaaa5412d8250db735f2c4300190d1a889c5